sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -122,7 +122,7 @@ Other optional properties can be set such as verbosity and whether service loggi
 
 ### General
 
-When you interact with the Cognitive Language Services Conversations client library using the .NET SDK, errors returned by the service correspond to the same HTTP status codes returned for [REST API][conversationanalysis_rest_docs] requests.
+When you interact with the Cognitive Language Services Conversations client library using the .NET SDK, errors returned by the service correspond to the same HTTP status codes returned for REST API requests.
```

Comment:
Are there no REST docs we can link? Some customers find these useful.

Reply:
Not yet :/ , we reached out to the service team and they are still working on REST documentation for inference. We can add it when it's ready though